api/rds.proto

```diff
@@ -137,6 +137,16 @@ message RouteAction {
     WeightedCluster weighted_clusters = 3;
   }
 
+  enum ClusterNotFoundResponseCode {
+    // HTTP status code - 503 Service Unavailable.
+    SERVICE_UNAVAILABLE = 0;
+    // HTTP status code - 404 Not Found.
+    NOT_FOUND = 1;
+  }
+  // The HTTP status code to use when configured cluster is not found.
+  // The default response code is 503 Service Unavailable.
+  ClusterNotFoundResponseCode cluster_not_found_response_code = 20;
```

Comment:
Can you add a defined only enum constraint here?

Reply:
Done (here and in `RedirectAction`).